### ISSUE TITLE
Missing latin variables fonts

### ISSUE THREAD

**Issue by tnoppeney** [philipbelesky/inter-ui#19]
Hi,

the following snippet is currently not working due to missing files:

```scss
@use "inter-ui/variable" with (
  $inter-font-display: swap,
  $inter-font-path: 'inter-ui/Inter (web latin)'
);

@include variable.default;
```
```
Error: Can't resolve 'inter-ui/Inter (web latin)/Inter-roman.var.woff2'
```
The `Inter-roman.var.woff2` and `Inter-italic.var.woff2` files are only available in the `Inter (web)` folder. Is that intentional?

Thank you!

**Comment by philipbelesky** [comment]
Hi, thanks for this report. This was not intentional and something I missed when the PR for the `@use` setups were merged in. This should be fixed in `9941d4058859a4e2115d0799c59302f6f4ef4a53`.

If possible, it would be great if you could check those new `var` files for anything - I presume the automatic subsetting works works OK with variable fonts, but I'm not well-placed to test that myself atm.

**Comment by philipbelesky** [comment]
Ok, hopefully this is resolved in the just-released 3.15.1
